plugin.xml

```diff
@@ -37,13 +37,13 @@
 
     <platform name="android">
         <source-file src="src/android/StatusBar.java" target-dir="src/org/apache/cordova/statusbar" />
+        <source-file src="src/android/ActivityAssistant.java" target-dir="src/org/apache/cordova/statusbar" />
 
         <config-file target="res/xml/config.xml" parent="/*">
             <feature name="StatusBar">
                 <param name="android-package" value="org.apache.cordova.statusbar.StatusBar" />
                 <param name="onload" value="true" />
             </feature>
-            <preference name="StatusBarOverlaysWebView" value="true" />
```

Comment:
Does this not change the default behaviour?

Reply:
This property was added in config.xml in version 3.0.0 in Apache's repository.
In our code, the default value is false. So leaving this line would add a breakage change.
`doOverlay = preferences.getBoolean("StatusBarOverlaysWebView", false);`
Wouldn't that be a problem?